ctapipe/reco/sklearn.py

```diff
@@ -777,13 +778,34 @@ def predict_table(self, key, table: Table) -> Dict[ReconstructionProperty, Table
         fov_lon = table["hillas_fov_lon"].quantity + disp * np.cos(psi)
         fov_lat = table["hillas_fov_lat"].quantity + disp * np.sin(psi)
 
-        # FIXME: Assume constant and parallel pointing for each run
-        self.log.warning("Assuming constant and parallel pointing for each run")
+        self.log.warning("FIXME: Assuming constant and parallel pointing for each run")
+        if np.all(table["subarray_pointing_frame"] is CoordinateFrameType.ALTAZ):
+            pointing_alt = table["subarray_pointing_lat"]
+            pointing_az = table["subarray_pointing_lon"]
+        elif np.all(table["subarray_pointing_frame"] is CoordinateFrameType.ICRS):
+            pointing_altaz = SkyCoord(
+                ra=table["subarray_pointing_lon"],
+                dec=table["subarray_pointing_lat"],
+                frame="icrs",
+            ).transform_to(AltAz())
```

Comment:
This also won't work. Transformation from ICRS to the AltAz system needs location and observation time